pkg/util/runtime/features.go

```diff
@@ -34,6 +34,9 @@ const (
 	// FeaturePlayerTracking is a feature flag to enable/disable player tracking features.
 	FeaturePlayerTracking Feature = "PlayerTracking"
 
+	// FeatureMTLSEnabled is a feature flag to enable/disable mTLS in the allocator.
+	FeatureMTLSEnabled Feature = "MTLSEnabled"
```

Comment:
Disabling mtls is a new feature. We should also clarify that this is for allocator service. So, something like "AllocatorMTLSDisabled"?

Reply:
I agree. The default value of a feature flag should start with false, so changing this from Enable -> Disable will make a lot more sense.

Reply:
Makes sense to me. Fixed!